docs/topics/context-parameters.md

```diff
@@ -0,0 +1,125 @@
+[//]: # (title: Context parameters)
+
+<primary-label ref="experimental-general"/>
+
+> Context parameters replace an older experimental feature called [context receivers](whatsnew1620.md#prototype-of-context-receivers-for-kotlin-jvm).
+> To migrate from context receivers to context
+> parameters, you can use assisted support in IntelliJ IDEA, as described in
+> the related [blog post](https://blog.jetbrains.com/kotlin/2025/04/update-on-context-parameters/).
+>
+{style="tip"}
+
+Context parameters allow functions and properties to declare dependencies that are implicitly available in the
+surrounding context.
+
+With context parameters, you don't need to manually pass around values, such as services or dependencies, that are shared and rarely change across sets of function calls.
+
+Context parameters are not introduced as receivers in the body of a function. As a result, you need to use the name of the context parameters to access their members, unlike with context receivers, where the context is implicitly available.
```

Comment:
I'm sorry to raise this again, but I really think that the documentation should **not** be written from the point of view of somebody who know about context receivers. Because if you don't know about them, why would you expect them to be introduced as receivers? (since they are just parameters).

My suggestion is to drop this paragraph entirely, and move most of its content to the note above (I'll make a suggestion).